doc/modules/cross_validation.rst

```diff
@@ -235,6 +235,20 @@ Here is an example of ``cross_validate`` using a single metric::
     >>> sorted(scores.keys())
     ['estimator', 'fit_time', 'score_time', 'test_score', 'train_score']
 
+Cross-validation significance evaluation
+----------------------------------------
+
+Significance of cross validation scores can be evaluated using
```

Comment:
```suggestion
Significance of cross validation scores can be evaluated using the
```

Reply:
@vmanisha committed the suggested changes to https://github.com/scikit-learn/scikit-learn/pull/14769